plugins/woocommerce/src/Utilities/OrderUtil.php

```diff
@@ -185,4 +184,48 @@ public static function get_table_for_orders() {
 	public static function get_table_for_order_meta() {
 		return wc_get_container()->get( COTMigrationUtil::class )->get_table_for_order_meta();
 	}
+
+	/**
+	 * Counts number of orders of a given type.
+	 *
+	 * @since 8.7.0
+	 *
+	 * @param string $order_type Order type.
+	 * @return array<string,int> Array of order counts indexed by order type.
+	 */
+	public static function get_count_for_type( $order_type ) {
```

Comment:
This method can probably use unit test.

Reply:
Unit test added in 810b427ede4fd40f203c909e81d6e0e3dca6b474.